cloudx/handler.go

```diff
@@ -231,6 +231,30 @@ func (h *SnakeCharmer) EnsureContext() (*AuthContext, error) {
 	}
 
 	if len(c.SessionToken) > 0 {
+		client, err := newKratosClient()
+		if err != nil {
+			return nil, err
+		}
+		_, resp, err := client.V0alpha2Api.ToSession(h.ctx).XSessionToken(c.SessionToken).Execute()
+		if resp.StatusCode == http.StatusUnauthorized || err != nil {
```

Comment:
If `err != nil`, then this will panic because `resp` can be `nil`.

Reply:
I think that checking the session for the `nil` value would be better in that case.